dags/dim_active_users_aggregates.py

```diff
@@ -0,0 +1,92 @@
+"""
+DAG for running simple cross-check for product usage KPI metrics.
+
+Part of DENG-750 - first draft at how we could be executing a simple cross-check
+for tables used for product usage KPI metrics.
+
+This is WIP, and it meant for testing only. However, the failures recorded by
+this DAG are important and the standatd triage process should be followed
+as long as this DAG is active.
+"""
+
+
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.dummy import DummyOperator
+from airflow.sensors.external_task import ExternalTaskSensor
+from operators.gcp_container_operator import GKEPodOperator
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.tags import Tag
+
+default_args = {
+    "owner": "kignasiak@mozilla.com",
+    "start_date": datetime(2023, 3, 20),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "kignasiak@mozilla.com",
+        "mreid@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "retries": 0,
+}
+
+TAGS = [
+    "repo/telemetry-airflow",
+    Tag.ImpactTier.tier_1,
+]
+IMAGE = "gcr.io/moz-fx-data-airflow-prod-88e0/dim:latest"
+
+APP_NAME_POSTFIX = "active_users_aggregates"
+APPS = ("desktop",)
+
+PROJECT_ID = "moz-fx-data-shared-prod"
+EXEC_DATE = "{{{{ macros.ds_add(ds, -1) }}}}"
+
+with DAG(
+    "dim_active_users_aggregates",
+    default_args=default_args,
+    schedule_interval="30 4 * * *",
+    tags=TAGS,
+    doc_md=__doc__,
+) as dag:
+
+    run_all = DummyOperator(
+        task_id="run_all",
+    )
+
+    for app_name in APPS:
+        dataset = f"firefox_{app_name}" if app_name == "desktop" else app_name
+
+        wait_for_aggregates = ExternalTaskSensor(
+            task_id=f"wait_for_{app_name}",
+            external_dag_id="bqetl_analytics_aggregations",
+            external_task_id=f"{dataset}_active_users_aggregates",
+            execution_delta=timedelta(seconds=3600),
+            check_existence=True,
+            mode="reschedule",
+            allowed_states=ALLOWED_STATES,
+            failed_states=FAILED_STATES,
+            pool="DATA_ENG_EXTERNALTASKSENSOR",
+        )
+
+        dim_check = GKEPodOperator(
+            task_id=f"{app_name}_{APP_NAME_POSTFIX}_dim_check",
+            name=f"{app_name}_{APP_NAME_POSTFIX}_dim_check",
+            image=IMAGE,
+            arguments=[
+                "run",
+                "--fail_process_on_failure",
+                f"--project_id={PROJECT_ID}",
+                f"--dataset={dataset}",
```

Comment:
If you decide to use app_name
```suggestion
                f"--dataset={app_name}",
```